ballerina/http_service.bal

```diff
@@ -93,6 +93,7 @@ isolated service class HttpService {
                 if self.isSubscriptionValidationDeniedAvailable {
                     processSubscriptionDenial(caller, response, params, self.adaptor);
                 } else {
+                    log:printWarn("Subscription is denied by the hub");
```

Comment:
I think it should be an error.

Reply:
Fixed with [6e4a022b402367cf84c6505660b2411db98fddf8](https://github.com/ballerina-platform/module-ballerina-websub/pull/468/commits/6e4a022b402367cf84c6505660b2411db98fddf8)